documentation/tutorial/installing_the_diagnostics_tools.md

```diff
@@ -27,8 +25,6 @@ In order to generate core dumps for .net core apps, you can use the dotnet-dump
 > dotnet tool install --global dotnet-dump --version 1.0.4-preview6.19311.1
 > ```
 
-Please note, the dotnet-trace tool is version independent but as of right now (preview 5) it requires the 2.1 runtime to be installed to work. This will be fixed moving forward.
```

Comment:
Should we also update https://github.com/dotnet/diagnostics/blob/master/documentation/dotnet-trace-instructions.md#installing-dotnet-trace to reflect this change? (about needing 2.1 SDK to be installed)

Reply:
Done